owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -54,6 +55,12 @@ open class FolderPickerActivity : FileActivity(),
 
     private lateinit var binding: FilesFolderPickerBinding
 
+    private var isInSpacesScreen: Boolean = false
+
+    private lateinit var menuItem: MenuItem
+
+    private lateinit var searchView: SearchView
```

Comment:
Same as in the other: these 2 properties are just used in 1 place, they don't need to be properties. You can remove them. Also, in `ReceiveExternalFilesActivity` you can remove `mSearchView` as an attribute and transform it into a local variable 👍

Reply:
Yeah. I have changed those two propertis and now are local values in `onCreateOptionsMenu` function.